src/DonationForms/resources/registrars/templates/layouts/HeaderImage.tsx

```diff
@@ -3,6 +3,17 @@ import {HeaderImageProps} from '@givewp/forms/propTypes';
 /**
  * @since 3.5.0
  */
-export default function HeaderImage({url, alt}: HeaderImageProps) {
-    return <img src={url} alt={alt} />;
+export default function HeaderImage({url, alt, color, opacity}: HeaderImageProps) {
+    // @ts-ignore
```

Comment:
Can we try to appease typescript without the ignore?

Reply:
Yes, this was actually resolved by adding the React.CSSProperties. I removed the @ts-ignore flag. 
Resolved: da19cf635bc405ee85b11b5919e26dc0d9c8721e